.mockery.yaml

```diff
@@ -0,0 +1,15 @@
+quiet: False
+inpackage: True
+disable-version-string: True
+with-expecter: True
+
+filename: "{{ trimSuffix .InterfaceFile \".go\" | base | lower }}_mock_test.go"
+dir: "{{ .InterfaceDir }}"
+mockname: "Mock{{ .InterfaceName }}"
+outpkg: "{{ .PackageName }}"
+
+packages:
+  github.com/kong/gateway-operator/controller/konnect/ops:
+    interfaces:
+      ControlPlaneSDK:
+      ServicesSDK:
```

Comment:
Nit: we might as well already generate mocks for other interfaces that are added in this PR (`ConsumerGroupSDK`, `ConsumersSDK`, `RoutesSDK`).

Reply:
I'll add mocks of all used SDKs in the following PR to refactor the usage of SDK.